operators/derive/to_es.py

```diff
@@ -26,7 +26,7 @@ def _convert_type(cls, schema_type, field, prefix):
 def data_api_es_flow():
     return DF.Flow(
         DF.load(f'{settings.DATA_DUMP_DIR}/card_data/datapackage.json'),
-        DF.set_type('card_id', type='string', **{'es:keyword': True}),
+        DF.set_type('card_id', **{'es:keyword': True}),
         DF.set_type('branch_id', type='string', **{'es:keyword': True}),
```

Comment:
the comment about setting the types applies to all `set_type` here :)

Reply:
:)